pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -672,6 +711,34 @@ func (o *SyncOptions) enableSyncerForWorkspace(ctx context.Context, config *rest
 	return string(saTokenBytes), syncerID, syncTarget, nil
 }
 
+func mergeLatestResourceSchema(apiExport *apisv1alpha1.APIExport, resourceToSync []string) (*apisv1alpha1.APIExport, bool) {
+	desiredResourceGroup := sets.NewString()
+	var modified bool
+	for _, schema := range apiExport.Spec.LatestResourceSchemas {
+		comps := strings.SplitN(schema, ".", 3)
```

Comment:
Why can we assume that these schema names have this format? Can we use some other (strongly-typed) data instead of strings here?

Reply:
Will this always handle multi-part group names correctly?

Reply:
change to use a func that return revision and groupResource.